schema/partner_artist.js

```diff
@@ -0,0 +1,72 @@
+import gravity from '../lib/loaders/gravity';
+import Partner from './partner';
+import Artist from './artist/index';
+import numeral from './fields/numeral';
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLNonNull,
+  GraphQLBoolean,
+} from 'graphql';
+
+const PartnerArtistType = new GraphQLObjectType({
+  name: 'PartnerArtist',
+  fields: () => ({
+
+    counts: {
+      type: new GraphQLObjectType({
+        name: 'PartnerArtistCounts',
+        fields: {
+          artworks: numeral(({ published_artworks_count }) =>
+            published_artworks_count),
+          for_sale_artworks: numeral(({ published_for_sale_artworks_count }) =>
+            published_for_sale_artworks_count),
+        },
+      }),
+      resolve: (partner_artist) => partner_artist,
+    },
+    id: {
+      type: GraphQLString,
+    },
+    is_display_on_partner_profile: {
+      type: GraphQLBoolean,
```

Comment:
I'm saying `is_display_on_partner_profile` isn't a thing anywhere, the JSON response from gravity is `display_on_partner_profile`.

If you want to follow the `is_` pattern for making these fields available in the schema, don't you need to resolve those with `display_on_partner_profile`?

Something like:

```js
is_display_on_partner_profile: {
  type: GraphQLBoolean,
   resolve: ({ display_on_partner_profile }) => display_on_partner_profile,
},
```

Right now theres no `resolve`, so wouldn't this just return null since the field (as named with the `is_` prefix) doesn't exist in the JSON response for a partner artist?